tests/cockroachdb_import_connect.robot

```diff
@@ -4,11 +4,12 @@ Metadata            Version    0.0.1
 
 Library             SeleniumLibrary
 Resource            ../resources/keywords/deploy_application.resource
+Resource            ../resources/keywords/suite_teardown.resource
 
 Suite Setup         Set Library Search Order    SeleniumLibrary
 Suite Teardown      Tear Down The Test Suite
 Test Setup          Given The Browser Is On Openshift Home Screen
-Test Teardown       Close Browser
+Test Teardown       Tear Down The Test
```

Comment:
IMHO we require "Test Down The Test" can be included for Admin and Developer provision test suites, where the import connect test suites might not need this step as we are not provisioning any database instances.